### ISSUE TITLE
Add Repository

### ISSUE THREAD

**Issue by davidnav19** [google-developer-training/basic-android-kotlin-compose-training-mars-photos#39]
**URL of codelab:**
https://developer.android.com/codelabs/basic-android-kotlin-compose-add-repository#8

**Specify the language of the codelab if it is not English:**

**In which task and step of the codelab can this issue be found?**
[9. Write a repository test]
Create an instance of the NetworkMarsPhotosRepository and pass the FakeMarsApiService as the marsApiService parameter.

**Describe the problem**
Following the codelab from previous steps, it seems that the NetworkMarsPhotosRepository was previously called DefaultMarsPhotosRepository.
In other words, in step 9, is the first time that the word NetworkMarsPhotosRepository is mentioned in the whole codelab

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_
_API version of the emulator:_

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._

**Comment by osuleymanova** [comment]
Duplicate of #34
